src/app/dashboard/accounts/[guid]/page.tsx

```diff
@@ -90,7 +92,10 @@ export default function AccountPage({ params }: AccountPageProps): JSX.Element {
         <div className="col-span-2 col-end-13 justify-self-end">
           <AddTransactionButton
             account={account}
-            onSave={() => mutate(`/api/splits/${params.guid}`)}
+            onSave={() => {
+              save();
```

Comment:
simple! it basically exports the DB and uses the storage instance to upload the data.